gplaycli/gplaycli.py

```diff
@@ -260,9 +260,10 @@ def prepare_download_updates(self, list_apks_to_update):
             for packagename, filename, apk_version_code, store_version_code in list_apks_to_update:
                 message += u"\n%s Version : %s -> %s" % (filename, apk_version_code, store_version_code)
                 list_of_packages_to_download.append([packagename, filename])
-            message += "\n\nDo you agree?"
+            message += "\n"
             print message
             if not self.yes:
+                print("\nDo you agree?")
```

Comment:
Please use print keyword instead of print function, to be coherent with the rest of the package.